doc/ref_model/chapters/chapter03.md

```diff
@@ -541,35 +541,54 @@ SyncE architecture minimally requires replacement of the internal clock of the E
 The following principles apply to Storage scope for the Reference Model, Reference Architectures, Reference Implementations and Reference Conformance test suites:
 * Abstraction: A standardized storage abstraction layer between the Virtualisation Layers and the Storage Physical Resources Layer that hides (or abstracts) the details of the Storage Physical resources from the Virtualisation Layers.
 * Agnosticism: Define Storage subsystem concepts and models that can provide various storage types and performance requirements (more in Virtual Resources [3.2.1.3 Storage](#3.2.1.3)).
 * Automation: Enable end-to-end automation, from Physical Storage installation and provisioning to automation of workloads (VNF/CNF) onboarding.
 * Openness: All storage is based on open source or standardized APIs (North Bound Interfaces (NBI) and South Bound Interfaces (SBI)) and should enable integration of storage components such as Software Defined Storage controllers.
 * Scalability: Storage model enables scalability to enable small up to large deployments.
 * Workload agnostic: Storage model can provide storage functionality to any type of workloads, including: tenant VNF, CNF and Infrastructure Managment whether this is via BareMetal or Virtualised Deployments.
-* Operationally Ammenable: The storage must be ammenable to consistent set of operational processes for: Non-Disruptive Capacity Expansion and Contraction, Backup/Restoration and Archive and Performance Management. Where applicable (Backup/Restoration/Archive) these processes should also be able to be provided to tenants for there own delegated management.
+* Operationally Amenable: The storage must be amenable to consistent set of operational processes for: Non-Disruptive Capacity Expansion and Contraction, Backup/Restoration and Archive and Performance Management. Where applicable (examples are: Backup/Restoration/Archive) these processes should also be able to be provided to tenants for their own delegated management.
+* Security Policy Amenable: The storage sub-systems must be ammenable to policy based security controls covering areas such as: Encryption for Data at Rest / In Flight, Delegated Tenant Security Policy Management, Platform Management Security Policy Override, Secure Erase on Device Removal and others
 * Future proof: Storage model is extendible to support known and emerging technology trends covering spectrum of memory-storage technologies including Software Defined Storage with mix of SATA- and NVMe-based SSDs, DRAM and Persistent Memory, integrated for multi-clouds, and Edge related technologies.
 
+NOTE: In a market assessemt of storage (for a "Ceph" based solution) costs done in 2019, the following comparative costs where found when trying to meet the different Storage Types needs across varies performance levels, for consistent usable capacity:
```

Comment:
Is the RM really the place for a cost analysis? I don't think we should include this part.

Reply:
Hi @ulikleber ,
it was added to give some level of quantification of "Cost" as part of typical Cost vs Performance consideration, based on feedback from @pgoyal01 .
As hard costs are pretty much out of date the from the day provided it was agreed to put in some indicative (Comparative) Cost to provide guidance to practitioner.
I could remove but then model became purely theoretical or alternately could move somewhere else...
It is team consensus call to my mind, I am ok to take action as agreed.

Reply:
@jbhartley, @ulikleber: There are good arguments for both sides, it is an important info but we have never done such an analysis, but it is only a Note, but how to keep such an info up-to-date?  I suggest asking this question at the RM meeting this Wednesday.  Obviously  we do not want this Note to become a blocking issue...

Reply:
@walterkozlowski @ulikleber @pgoyal01 
Given timing I have removed this note as we can resolve with next release when there is less urgency and greater time to discuss and align.